tests/test_cluster_ref_key.hh

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file   test_cluster_ref_key.hh
+ *
+ * @author Alexander Goscinski <alexander.goscinski@epfl.ch>
+ * @date   10 Thu 2019
+ *
+ * @brief  tests for the cluster ref key layer computation utilities
+ *
+ * Copyright  2019 Alexander Goscinski, COSMO (EPFL), LAMMM (EPFL)
+ *
+ * Rascal is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3, or (at
+ * your option) any later version.
+ *
+ * Rascal is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software; see the file LICENSE. If not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ */
+
+
+#ifndef TESTS_TEST_CLUSTER_REF_KEY_HH_
+#define TESTS_TEST_CLUSTER_REF_KEY_HH_
+
+#include "tests.hh"
+#include <tuple>
+#include "structure_managers/cluster_ref_key.hh"
+
+/**
+ * Prints the index sequence of an std::index_sequence type by recursively
+ * printing the head element.
+ *
+ * @param _ the std::index_sequnce to print
+ */
+namespace rascal {
+  struct LayerFixture {
+    // {1}
+    using order_one_seq = std::index_sequence<1>;
+    // {1,1}
+    using order_two_seq = std::index_sequence<1, 1>;
+    // {1,1,0}
+    using order_three_seq = std::index_sequence<1, 1, 0>;
+  };
+
+  /**
+   * Prints the index sequence `Ints` defined in the std::index_sequence type of
+   * the parameter `sequence` by recursively printing the head element.
+   *
+   * @param sequence sequence to be printed
+   */
+  template<size_t... Ints>
+  void print_index_sequence(std::index_sequence<Ints...> /*sequence*/) {
```

Comment:
Would I be right in declaring this a helper-function?
If so, could you add that to the description.